docs/_static/css/theme_overrides.css

```diff
@@ -0,0 +1,19 @@
+@import 'theme.css';
+
+@media screen and (min-width:1100px) {
+    .wy-nav-content {
+        max-width: 1000px
+    }
+}
+
+.rst-content code,.rst-content tt {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace
+}
+
+.rst-content code.literal {
+    color: #595959
+}
+
+.rst-content .viewcode-back,.rst-content .viewcode-link,.rst-content a code.xref {
+    color: #007020
+}
```

Comment:
I’d prefer to just keep the default RTD theme unless there’s a good reason to change it. These changes seem rather subjective for the most part.

Reply:
The colors of the non-stdlib things being red is super obnoxious (i.e. things like `None` and `True` being bright red everywhere on the page) and the stuff that is actually clickable (cross-refs) is not made apparent so I changed it to the same color green as is in the `[source]` plug-in that comes with Sphinx. See <https://lvsfunc.readthedocs.io/en/latest/#lvsfunc.scale.test_descale> for cross-refs and things like `int` just being the same color although you can click on only the cross-refs. And see <https://lvsfunc.readthedocs.io/en/latest/#lvsfunc.scale.test_descale> for the red text, which btw appears a lot more on vsutil's docs than it does here.

The max-width is something stux brought up in Discord way back when the autosummary stuff was being hidden with a scroll-bar, and 1000 seems to be a happy medium between the default 800px and the max on a normal 16:9 1080p screen, which would be like 1600-1700-ish.

I'd be willing to get rid of the width thing, and maybe the font stack, but the colors of the default theme for in-line code is seriously terrible and lacking.

Reply:
Default:
![image](https://user-images.githubusercontent.com/10972040/88613063-ad383f00-d05a-11ea-87f3-e775922fe0a7.png)
Custom CSS:
![image](https://user-images.githubusercontent.com/10972040/88613085-bb865b00-d05a-11ea-9744-4bacaf9bf9fc.png)

Reply:
> the stuff that is actually clickable (cross-refs) is not made apparent

Has that ever been brought up in an issue upstream? Changing this sounds like something most users would benefit from.

I’d like to get rid of the font overrides (because that’s really just personal preference), but I see your point for the rest. Good changes.